jesusdavid24/front-app-restaurant
# Feature/booking

## Feature - Booking

### Description

This pull request introduces a new feature that allows capturing user input within a form to generate a booking. The implementation utilizes Redux to manage the booking state and communicates with the API through a service to persist the booking data in the database.

### Changes Made

- Implemented a form to capture user input for creating a booking.
- Integrated Redux for state management, using a reducer to handle the booking state.
- Developed a service to communicate with the API for storing booking data in the database.

### How to Test

1. Navigate to the booking section of the application.
2. Fill out the required information in the booking form.
3. Submit the form and verify that the booking is correctly saved in the database.
   
### Screenshots

![image](https://github.com/jesusdavid24/front-app-restaurant/assets/132521237/2620c084-cd7a-4b92-91d9-a533e8885baf)
<br/>
![image](https://github.com/jesusdavid24/front-app-restaurant/assets/132521237/3f6907ad-82e7-4059-885c-f4e1cf2e0a05)

### Checklist

- [x] Implemented the user input form for booking.
- [x] Utilized Redux for state management.
- [x] Developed a service for API communication and data storage.
- [x] Verified that the booking data is correctly persisted in the database.
